### PR TITLE
dev-util/packer: update to 1.4.0

### PR DESCRIPTION
**Pull request by dmolik**
Package-Manager: Portage-2.3.62, Repoman-2.3.12
Signed-off-by: Dan Molik <dan@danmolik.com>
Tested-by: Dan Molik <dan@danmolik.com>

### CODE REVIEW

File: dev-util/packer/packer-1.4.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (gyakovlev) [inline dev-util/packer/packer-1.4.0.ebuild]: can be bumped to 7

File: dev-util/packer/packer-1.4.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+EGO_PN="github.com/hashicorp/packer"
+SRC_URI="https://github.com/hashicorp/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+DEPEND=">=dev-lang/go-1.11.0"
+
+DOCS="README.md CHANGELOG.md"
```
Comment (gyakovlev) [inline dev-util/packer/packer-1.4.0.ebuild]: please make it an array
`DOCS=( README.md CHANGELOG.md )`

Comment (dmolik) [reply dev-util/packer/packer-1.4.0.ebuild]: Done

File: dev-util/packer/packer-1.4.0.ebuild (hunk 1 above)
Comment (dmolik) [reply dev-util/packer/packer-1.4.0.ebuild]: Done, and tested

**Pull request closed without merging**